styleguide/derek/_partials/_mainnav.ejs

```diff
@@ -109,10 +109,10 @@
               <li class="navbar-topItem">
                 <span class="navbar-topLink dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Solutions</span>
                 <ul class="dropdown-menu navbar-dropDownMenu">
-                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLink active">Managed Offerings</a></li>
+                  <li class="navbar-dropDownItem"><label column="1" columntitle="1" class="navbar-dropDownLabel">Managed Offerings</a></li>
```

Comment:
omg i am so sorry. I wasn't paying nearly as much attention as I should have. Fixing it right now.